app/Helpers/Helpers.php

```diff
@@ -626,6 +626,9 @@ function formatFilesetMeta($fileset)
                     $fileset[$metadata['name']] = $metadata['description'];
                 }
             }
+            // December 2022, this is temporary to support older versions of 5fish applications.
+            // Revisit in one year after discussing with James Thompson.
```

Comment:
My mistake in giving you the wrong name. Please change to James Thomas (mailto:jamesthomas@globalrecordings.net)

Reply:
@bradflood , I just updated it. The comment is:

// December 2022, this is temporary to support older versions of 5fish applications.
// Revisit in one year after discussing with James Thomas mailto:jamesthomas@globalrecordings.net